Update repository templates and documents. (#115)
darwinai/covidnet_ui: README.md

<!-- TO DO: include instructions to upload using particular packages specified by fnndsc -->

> ⚠️ See the **covidnet_integration** repository README.md for more details on how to add sample DICOMs to the COVID-Net UI.
> ```shell
> cd <path to the covidnet_integration repo>
> ./make.sh
> docker run --network host -v "$PWD/images/WithProtocolName/COVID-19:/images" covidnet_integration upload_swift_notify_cube.py --imageDir /images
> docker run --network host -v "$PWD/images/WithProtocolName/Normal:/images" covidnet_integration upload_swift_notify_cube.py --imageDir /images
> docker run --network host -v "$PWD/images/WithProtocolName/Pneumonia:/images" covidnet_integration upload_swift_notify_cube.py --imageDir /images
> ```

Reviewer: ```suggestion
⚠️ See the **covidnet_integration** repository README.md for more details about how to add sample DICOMs to the COVID-Net UI.
```shell
cd <path to the covidnet_integration repo>
./make.sh
docker run --network host -v "$PWD/images/WithProtocolName/COVID-19:/images" covidnet_integration upload_swift_notify_cube.py --imageDir /images
docker run --network host -v "$PWD/images/WithProtocolName/Normal:/images" covidnet_integration upload_swift_notify_cube.py --imageDir /images
docker run --network host -v "$PWD/images/WithProtocolName/Pneumonia:/images" covidnet_integration upload_swift_notify_cube.py --imageDir /images
```
```
